_tour/implicit-conversions.md

```diff
@@ -10,46 +10,33 @@ previous-page: implicit-parameters
 redirect_from: "/tutorials/tour/implicit-conversions.html"
 ---
 
-An implicit conversion from type `S` to type `T` is defined by an implicit value which has function type `S => T`, or by an implicit method convertible to a value of that type.
+{% tabs implicit-conversion-defn class=tabs-scala-version %}
+{% tab 'Scala 2' %}
+In Scala 2, an implicit conversion from type `S` to type `T` is defined by an [implicit value]({% link _tour/implicit-parameters.md %}) which has function type `S => T`, or by an implicit method convertible to a value of that type.
+{% endtab %}
+{% tab 'Scala 3' %}
+In Scala 3, an implicit conversion from type `S` to type `T` is defined by a [given instance]({% link _tour/implicit-parameters.md %}) which has type `scala.Conversion[S, T]`, or by an implicit method which can be eta-expanded to the function type `S => T`.
```

Comment:
I think in Scala 3 the support for implicit methods (and implicit values of type functions) is there only for compatibility with Scala 2, no? I would rather say that in Scala 3 an implicit conversion is defined by a `given` instance of type `Conversion`. And then, say that for compatibility with Scala 2, they can also be defined as `implicit` methods (and tell the readers that they will find more information on the Scala 2 tab).

Reply:
that's a good suggestion, I will follow up with that